.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+language: minimal
+
+services: docker
+
+script:
+    - make all-tests
```

Comment:
It'd be useful to have an `install` step like https://github.com/libero/browser/blob/a88bdd99bb8634967f21aac36d16e8ebbc77d4cd/.travis.yml#L12-L13 to help reduce timeout errors.

Reply:
added install step.